include_core/omrport.h

```diff
@@ -2483,6 +2485,10 @@ typedef struct OMRPortLibrary {
 	int32_t (*port_startup_library)(struct OMRPortLibrary *portLibrary) ;
 	/** see @ref omrport.c::omrport_create_library "omrport_create_library"*/
 	int32_t (*port_create_library)(struct OMRPortLibrary *portLibrary, uintptr_t size) ;
+#if defined(OMR_ENV_DATA64)
+	/** see @ref omrport.c::omrport_copy_suballocator_heap_globals "omrport_copy_suballocator_heap_globals"*/
```

Comment:
nit: there should be a space before `*/`.

Reply:
Since these only affect comments, I will wait to push until after the builds complete.